cmd/plugin/cmd/init_test.go

```diff
@@ -375,100 +372,6 @@ func generateCAPIOperatorDeployment(name, namespace string) *appsv1.Deployment {
 	}
 }
```

Comment:
One question - this test removed, is it covered by upgrade plan tests? Shouldn't we leave both for clarity?